app/mailers/canonical_pending_transaction_mailer.rb

```diff
@@ -11,6 +11,11 @@ def notify_approved
       id: @cpt.local_hcb_code.hashid,
       s: @cpt.local_hcb_code.signed_id(expires_in: 2.weeks, purpose: :receipt_upload)
     )
+    @receipt_status_url = Rails.application.routes.url_helpers.receipt_status_hcb_code_url(
+      id: @cpt.local_hcb_code.hashid,
+      format: :svg,
+      s: @cpt.local_hcb_code.signed_id(purpose: :receipt_status)
+    )
```

Comment:
Is there a reason we can't just include this in the view code? Not sure why it needs an instance variable.

Reply:
moved! I had put it in the mailer since I saw the upload URL was there, but it works in the view as well